### ISSUE TITLE
(Self-Voicing) Minor content updates for GFLB's Read-Me buttons

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#236]
Inspired by changes needed for the Sim Overview in John Travoltage, I decided to review the Sim Overview for this sim.

@jessegreenberg, for GFLB, I have done 3 things:

1. removed references to the Play Area and Control Area
2. made very minor wording changes to surrounding text to make things continue to flow, and
3. added a comma in the Hint content which hopefully will make the synthetic speech more understandable at higher speeds.

There are no changes to the content used for Current Details button.

**Sim Overview:** (added words marked in bold and deleted text not show)
>> Gravity Force Lab: Basics is an interactive sim. It changes as you play with it. 
>> **There are** two mass spheres, a blue sphere labeled mass 1 and a red sphere labeled mass 2. A force arrow starts at the center of each sphere and points directly at the opposite sphere.Spheres can be moved closer or farther from each other along a track. The mass of each sphere can be increased or decreased. Each sphere is held in place by a robot.
>> **In addition**, checkboxes change what things are shown and how they behave, and a button resets the sim.

**Hint Please!**
>> Change position, and mass of spheres to explore.

See the design doc for a direct comparison of new content and deleted content - [Content for Read Me Buttons](https://docs.google.com/document/d/1XFTD2HTcGCEOP2yQ9_2SkEeXmCceZYUCmNeyiJfcCHs/edit#heading=h.hvsl429ubed7)

**Comment by jessegreenberg** [comment]
Done in the above commits @terracoda, can you please review?

**Comment by terracoda** [comment]
I hear "Control Area" at the end of the Sim Overview. It must have got left in by accident.

Other than that, it sounds great. The hint sounds much better with the comma in place, too.

Back over to you @jessegreenberg,. You can close after "Control Area" is removed from the "Sim Overview"

**Comment by terracoda** [comment]
@jessegreenberg, we still need some changes to the read-me button content.

Hint is good.
Current Details are good.

**Sim Overview should be:**
Gravity Force Lab: Basics is an interactive sim. It changes as you play with it.
There are two mass spheres, a blue sphere labeled mass 1 and a red sphere labeled mass 2. A force arrow starts at the center of each sphere and points directly at the opposite sphere. Spheres can be moved closer or farther from each other along a track. The mass of each sphere can be increased or decreased. Each sphere is held in place by a robot.
In addition, checkboxes change what things are shown and how they behave, and a button resets the sim.

**Comment by terracoda** [comment]
@jessegreenberg, some content in the middle is missing. I am bolding that content for you:

"Gravity Force Lab: Basics is an interactive sim. It changes as you play with it.
There are two mass spheres, a blue sphere labeled mass 1 and a red sphere labeled mass 2. A force arrow starts at the center of each sphere and points directly at the opposite sphere. **Spheres can be moved closer or farther from each other along a track. The mass of each sphere can be increased or decreased.** Each sphere is held in place by a robot.
In addition, checkboxes change what things are shown and how they behave, and a button resets the sim."

Then there is an extra sentence at the end that needs to be removed:
"In the control area, checkboxes change what things are shown and how they behave, and a button resets the sim."

**Comment by jessegreenberg** [comment]
>  hear "Control Area" at the end of the Sim Overview. It must have got left in by accident.

Sorry, I think this was fixed in #239. I am not hearing it anymore.

> some content in the middle is missing. 

Thanks, should be added now after the above commit. Should be able to verify on master, can you please check it sounds OK?

**Comment by terracoda** [comment]
@jessegreenberg, you got it. It all sounds good now.
Thanks! Closing!
